state/protocol/badger/state.go

```diff
@@ -37,15 +37,15 @@ type State struct {
 		setups  storage.EpochSetups
 		commits storage.EpochCommits
 	}
-	globalParams             protocol.GlobalParams
+	params                   protocol.Params
 	protocolStateSnapshotsDB storage.ProtocolState
 	protocolState            protocol.MutableProtocolState
 	versionBeacons           storage.VersionBeacons
 
-	// rootHeight marks the cutoff of the history this node knows about. We cache it in the state
+	// finalizedRootHeight marks the cutoff of the history this node knows about. We cache it in the state
 	// because it cannot change over the lifecycle of a protocol state instance. It is frequently
 	// larger than the height of the root block of the spork, (also cached below as
-	// `sporkRootBlockHeight`), for instance if the node joined in an epoch after the last spork.
+	// `sporkRootBlockHeight`), for instance, if the node joined in an epoch after the last spork.
 	finalizedRootHeight uint64
 	// sealedRootHeight returns the root block that is sealed.
```

Comment:
```suggestion
	// sealedRootHeight returns the root block that is sealed. We cache it in
	// the state, because it cannot change over the lifecycle of a protocol state instance.
```